packages/pwa-kit-react-sdk/src/ssr/universal/utils.js

```diff
@@ -19,6 +19,9 @@ const onClient = typeof window !== 'undefined'
  * @returns {string}
  */
 export const getAssetUrl = (path) => {
+    if (/^\/?mobify\/bundle\//.test(path)) {
+        return path
+    }
```

Comment:
@vmarta why do we need this change? Have you tested this locally and remotely?

Reply:
I feel like we are not solving the root issue here. Do you know which assets have double path, and which ones do not? Do you have examples?

Reply:
@kevinxh It was these assets/chunks:
```html
<script async="" data-chunk="main" src="/mobify/bundle/development/mobify/bundle/development/vendor.js"></script>
<script async="" data-chunk="main" src="/mobify/bundle/development/mobify/bundle/development/main.js"></script>
<script async="" data-chunk="pages-home" src="/mobify/bundle/development/mobify/bundle/development/pages-home.js"></script>
```

Other static assets like images do not have the issue.

Reply:
The root issue is without adding the `publicPath`, *.hot-update.json requests would be made against the root '/' and thus resulting in 404. I wanted to update their urls to be more like this '/mobify/bundle/development/*.hot-update.json'.

Reply:
these looks like the loadable chunks

Reply:
i believe you can tell the webpack hot middleware where to grab the hot reloading assets, try this webpack entry `webpack-hot-middleware/client?path=/mobify/bundle/development`

Reply:
@kevinxh according to the [doc](https://github.com/webpack-contrib/webpack-hot-middleware#client), the `path` is meant for something else. It's meant for specific path to the stream url. By default, path is set to this: `webpack-hot-middleware/client?path=/__webpack_hmr`

Reply:
Oh turns out there's this webpack option: https://webpack.js.org/configuration/output/#outputhotupdatemainfilename